### ISSUE TITLE
Error "Unfortunately, Hacker's Keyboard has stopped."

### ISSUE THREAD

**Issue by GoogleCodeExporter** [klausw/hackerskeyboard#455]
```
[ Before posting bugs, please check
https://code.google.com/p/hackerskeyboard/wiki/FrequentlyAskedQuestions and
the existing bugs for known issues - my responses may be delayed since I
have very little time to work on this project. ]

What steps will reproduce the problem?
1. go to Language and input
2. select Hacker's Keyboard as the default input method

What is the expected behavior? What do you see instead?
Expected: lets me use Hacker's Keyboard
Instead: error msg "Unfortunately, Hacker's Keyboard has stopped."

What version of Hacker's Keyboard are you using, for example "1.37"? (See
"Debug" section at the bottom of the app's Settings menu.)
1.37

On what phone or tablet?
Samsung Galaxy S4 - Android version 4.4.2

If applicable, does this affect the 4-row or 5-row layout, or both? Which
language(s)?
Can't use the keyboard at all.

Please provide any additional information below.
I have used Hacker's Keyboard for several years. One of the best downloads! But 
now it would not work at all. I tried downloading/installing older versions, 
but get an error "can't install". HELP, please!!!
```

Original issue reported on code.google.com by `perr...@gmail.com` on 18 Feb 2015 at 9:20

**Comment by WayneBuckhanan** [comment]
Had the same issue. Sent a crash report or two.

Reinstalled from F-Droid to get a more recent version and immediately on activation as the default kb I got two "stopped" messages.

Realizing this only started when I had deleted entries in the user dictionary with User Dictionary Manager, I reinstalled UDM and cleared out all my custom dictionary entries (after exporting them) and now I'm back to being able to use HK! 
Huzzah!

So, either there is something weird about how UDM edits the user dictionary or there is an issue in parsing the dictionary after UDM has deleted entries.
